README.md

```diff
@@ -121,16 +137,16 @@ not required):
        "irods_default_resource" : "demoResc"
    }
    ```
-   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request.
+   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request. Be sure to set the *irods_default_resource*, this variable is not set when you create the file with *iinit* or when you copy it over from another user.
    
 2. As the user who runs the GridFTP server, try an `ils` icommand to verify that 
    the information set in the *irods_environment.json* are fine. If needed, perform 
    an `iinit` to authenticate the iRODS user. 
 
-2. Add the following lines to the GridFTP configuration file ( tipically 
-   *$GLOBUS_LOCATION/etc/gridftp.conf* ):
+3. Add the following lines to the GridFTP configuration file (typically 
+   *$GLOBUS_LOCATION/etc/gridftp.conf*):
    ```
-   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI"
+   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI/B2STAGE-GridFTP/"
```

Comment:
Hi Vlad, 

you are right, having the build directory as a sub-directory in the installation directory was certainly not the smartest move.
I tried the whole installation again with a fresh *iRODS_DSI* and checkout of the *B2STAGE-GridFTP*.
Thus, the folder structure looks now like this:
```
.
├── iRODS_DSI
├── B2STAGE-GridFTP
└── irods*.deb
```
The *setup.sh* defines the *iRODS_DSI* folder as destination directory. I did not do any further editing of the files provideed by this git repository.
After going through the installation process  again I find the *so* files in both folders, B2STAGE-GridFTP and iRODS_DSI. Is this how it should be?